cmd/kubelet/app/options/options.go

```diff
@@ -454,6 +450,7 @@ func AddKubeletConfigFlags(mainfs *pflag.FlagSet, c *kubeletconfig.KubeletConfig
 	fs.Var(componentconfig.IPVar{Val: &c.Address}, "address", "The IP address for the Kubelet to serve on (set to `0.0.0.0` for all IPv4 interfaces and `::` for all IPv6 interfaces)")
 	fs.Int32Var(&c.Port, "port", c.Port, "The port for the Kubelet to serve on.")
 	fs.Int32Var(&c.ReadOnlyPort, "read-only-port", c.ReadOnlyPort, "The read-only port for the Kubelet to serve on with no authentication/authorization (set to 0 to disable)")
+	fs.StringVar(&c.BootstrapCheckpointPath, "bootstrap-checkpoint-path", c.BootstrapCheckpointPath, "<Warning: Alpha feature> Path to the directory where the checkpoints are stored")
```

Comment:
is this graduating from a alpha level feature?

Reply:
From #61681:
>As --bootstrap-checkpoint-path is an alpha/experimental flag, the feature it configures must either be
feature-gated, or graduated from alpha/experimental status prior to the migration.

Reply:
As the structure of feature-gates, we have to new another field like enableBootstrapCheckpointPath in bool if we'd move it into the feature-gates, which seems quite redundant, right? That maybe we should wait until it graduates from alpha. @mtaufen 
Another thing, how do you think about the rest migrations such as --bootstrap-kubeconfig and others. If it's ok to start the job I'd like to do it right now~

Reply:
@tianshapjq Please do! I'll take all the help with the other migrations I can get :). 

A gate just for this flag would be somewhat redundant, but maybe a gate for the overall Pod checkpointing feature? I like being able to validate alpha config fields against corresponding feature gates...

@yujuhong how close is this feature to exiting alpha?

Reply:
Yes, if we move it to the versioned config API and it remains alpha, it should have a feature gate, and it should be an error to set associated fields without also turning the feature gate on.